README.md

```diff
@@ -457,24 +457,24 @@ So in decimal format will become: `411-514-3`. This is the device id of `Heatit
 
 #### Thermostats
 
-```js
+```yaml
```

Comment:
That's not yaml, it' JSON

Reply:
I know it's not `yaml`, but since `json` doesn't support comments and there's a comment in there, and  `json` is valid `yaml` I figured I would set it to `yaml` just so the syntax highlighting works properly with IDEs. I can change it to `json` and move the comments to outside the code block.

references:
https://stackoverflow.com/questions/21584985/what-valid-json-files-are-not-valid-yaml-1-1-files
https://en.wikipedia.org/wiki/YAML#Comparison_with_JSON